apps/web/src/app/[locale]/challenge/_components/comments/comment-input.tsx

```diff
@@ -32,6 +45,12 @@ export function CommentInput({ mode, onCancel, onChange, value, placeholder, onS
     }
 
     if (e.shiftKey && e.key === 'Enter') {
+      const success = singleFieldSchema.safeParse(form.getValues()).success;
```

Comment:
is there a way to just trigger a form submit?

Reply:
I'm not sure... it needs to return if the inputs aren't valid, so I think safeParse is necessary, right?